fastlane/lib/fastlane/fastlane_require.rb

```diff
@@ -39,21 +39,23 @@ def install_gem_if_needed(gem_name: nil, require_gem: true)
       end
 
       def gem_installed?(name, req = Gem::Requirement.default)
-        # We fork and try to load a gem with that name in the child process so we
-        # don't actually load anything we don't want to load
-        # This is just to test if the gem is already preinstalled, e.g. YAML
-        # See https://github.com/fastlane/fastlane/issues/6951
-        fork do
-          begin
-            require name
-          rescue LoadError
-            exit(1)
-          end
+        installed = Gem::Specification.any? { |s| s.name == name and req =~ s.version }
```

Comment:
Good call moving this up, that's probably a good idea anyway. Small nitpick, instead of the 

```
unless installed
  # ...
end
```

Maybe just do an early return

Reply:
`return installed if installed`?

Reply:
Yeah or just `return true if installed` (since installed can only be true anyway, and to make that explicit)